packages/docs/celo-gold-holder-guide/quick-start.md

```diff
@@ -261,7 +269,7 @@ Your locked cGLD votes should be displayed next to `pending` under `votes`.
 
 Your vote will apply starting at the next Validator Election, held once per day, and will continue to apply at each subsequent election until you change it.
 
-After that election has occurred, you will need to activate your vote. This will allow you to receive epoch rewards if in that election (or at any subsequent one, until you change your vote) the validator group for which you voted elected at least one validator.
+After that election has occurred, you will need to activate your vote. This will allow you to receive epoch rewards if in that election (or at any subsequent one, until you change your vote) the validator group for which you voted elected at least one validator. Rewards will get added to your votes for that group and will compound automatically.
```

Comment:
I noticed a problem with line 284 below: 
`Your locked cGLD votes should be displayed next to active under votes.`
The votes were actually still under `pending` when I did this, which makes sense cause when that command is run they haven't been activated yet

Reply:
good spot!